apps/cookbook/src/app/shared/showcase-properties/showcase-property.ts

```diff
@@ -6,9 +6,21 @@ export interface ShowcaseProperty {
   preserveInputValuesWhitespaces?: boolean;
 }
 
-export interface ShowCasePropertyColumns {
+export interface ShowcasePropertyColumns {
   Name?: string;
   Description?: string;
   Type?: string;
   Default?: string;
 }
+
+export const defaultEventColumns: ShowcasePropertyColumns = Object.freeze({
```

Comment:
Why do you need to freeze them? 🤔

Reply:
To prevent properties in `defaultEventColumns` from being changed/added/removed